apps/snitch_core/lib/core/data/model/store.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Snitch.Data.Model.Store do
```

Comment:
Let's not keep the name of settings tab as `store` the name suggests something else.